### PR TITLE
updates and saucelabs desktop support

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,57 +1,37 @@
...
-          curl -sSL "https://raw.githubusercontent.com/h5bp-bot/scripts/0.6.1/update_site_branch.sh" |
-              bash -s -- --commands "grunt copy:gh-pages" \
-                         --commit-message "Hey server, this content is for you! [skip ci]" \
-                         --directory "gh-pages" \
-                         --distribution-branch "gh-pages" \
-                         --source-branch "master";
-        fi
-      fi
-
+- python travis_after_all.py
```
Comment (ryanseddon) [inline .travis.yml]: Can you not format this?

File: .travis.yml [hunk 2]
```diff
@@ -1,57 +1,37 @@
...
+  \ if [ \"$BUILD_AGGREGATE_STATUS\" == \"others_succeeded\" ]; then \\\n\n    # Clean
+  up helper files\n    rm -rf travis_after_all.py .to_export_back && \\\n\n    # The
+  `update_site_branch.sh` script will run the build,\n    # and if that generates
+  changes, it will commit them to the\n    # `master` branch:\n    #\n    #   * ensuring
+  that the content from the `gh-pages/` directory\n    #     is always in sync with
+  the rest of the content\n    #\n    #   * removing the need to execute the build
+  step locally\n    #     everytime a change is made (especially in the case\n    #
+  \    of trivial changes such as typos)\n    #\n    # Note: The `update_site_branch.sh`
+  script will only run\n    # if the commit was made to the `master` branch.\n\n    curl
+  -sSL \"https://raw.githubusercontent.com/h5bp-bot/scripts/0.6.1/update_site_branch.sh\"
```
Comment (ryanseddon) [inline .travis.yml]: Can we have a local copy of this shell script?

File: .travis.yml (hunk 1 above)
Comment (patrickkettner) [inline .travis.yml]: you can, it just got all crappy when I added new keys via the travis gem

File: .travis.yml (hunk 2 above)
Comment (patrickkettner) [inline .travis.yml]: its about to be rewritten by @alrra to make it a lot simplier. I can make it local once that happens

**Comment by patrickkettner** [comment]
chasing ghosts in the saucelabs, but once it _does_ land...

coveralls.io badge at 100%

**Pull request closed without merging**